README.md

```diff
@@ -1,3 +1,45 @@
+<p align="center">
+  <img src="static/glow_logo_horiz_color.png" width="300px"/>
+</p>
+
+<p align="center">
+	An open-source toolkit for large-scale genomic analyes
+  <br/>
+  <a href="https://glow.readthedocs.io/en/latest/?badge=latest"><strong>Explore the docs »</strong></a>
+  <br/>
+  <br/>
+  <a href="https://github.com/projectglow/glow/issues">Issues</a>
+  ·
+  <a href="https://groups.google.com/forum/#!forum/proj-glow">Mailing list</a>
+  ·
+	<a href="https://join.slack.com/t/proj-glow/shared_invite/enQtNzkwNDE4MzMwMTk5LTE2M2JiMjQ1ZDgyYWNkZTFiY2QyYWE0NGI2YWY3ODY3NmEwNmU5OGQzODcxMDBlYzY2YmYzOGM1YTcyYTRhYjA">Slack</a>
+</p>
+
+Glow is an open-source toolkit for working with genomic data at biobank-scale and beyond.
```

Comment:
Add personas who would use the tool, something like:
Glow is an open-source toolkit to enable bioinformatics and computational biology at biobank-scale and beyond.